Controllers/Backend/ConnectConfig.php

```diff
@@ -186,6 +187,32 @@ public function getExportAction()
             $exportConfigArray['exportPriceMode'] = array();
         }
 
+
+        //Resets the exportPriceMode and alternateDescriptionField
+        if (!array_key_exists('alternateDescriptionField', $exportConfigArray)
```

Comment:
Why we need to reset additional description field together with price type reset ?